apps/gateway/src/modules/weather/constants/weather-errors.const.ts

```diff
@@ -0,0 +1,3 @@
+export const WeatherErrors = {
+  INVALID_REQUEST: 'Invalid request',
```

Comment:
Are these errors intended for end users? If they’re meant for the API, it’s better to define them like INVALID_REQUEST: 'INVALID_REQUEST' so that the client can map them to user-friendly messages using an i18n library.